bin/tronctl

```diff
@@ -371,6 +384,25 @@ def move(args):
     yield request(urljoin(args.server, "/api/jobs"), data)
 
 
+def get_validated_max_parallel(max_parallel, max_parallel_limit):
+    if max_parallel <= 0:
+        raise SystemExit(f"The flag --max-parallel must be a positive integer.")
+    if max_parallel_limit <= 0:
+        raise SystemExit(f"The flag --max-parallel-limit must be a positive integer.")
+
+    if max_parallel_limit > LIMIT_MAX_PARALLEL_RUNS:
+        print(f"The flag --max-parallel-limit is overwritten to {max_parallel_limit} which is higher than " +
+              f"the suggested limit of {LIMIT_MAX_PARALLEL_RUNS}. If you haven't consulted the Tron team yet, " +
+              f"then please do so now to avoid overloading Tron with too many backfill jobs.")
```

Comment:
This warning of "If you haven't consulted the Tron team yet, ..." is a little strange considering that the code currently doesn't stop them. I would either remove the warning, or request user confirmation (like a "have you consulted the team" thing) before continuing.

Reply:
++ to requesting confirmation